profiles/deployment_context.dev-ppp

```diff
@@ -7,32 +7,32 @@ config_project_id         = "open-targets-partner-platform"
 config_deployment_regions = ["europe-west1"]
 
 // --- Elastic Search configuration --- //
-config_vm_elastic_search_data_volume_image = "ppp-20230628-1341-es"
+config_vm_elastic_search_data_volume_image = "ppp-20230922-0636-es"
 
 // --- Clickhouse configuration --- //
-config_vm_clickhouse_data_volume_image = "ppp-20230628-1341-ch"
+config_vm_clickhouse_data_volume_image = "ppp-20230922-0636-ch"
 
 // --- API configuration --- //
-config_vm_platform_api_image_version = "dev23.06.7"
+config_vm_platform_api_image_version = "dev23.09.7"
 config_vm_version_major              = "23"
-config_vm_version_minor              = "06"
-config_vm_version_patch              = "6"
+config_vm_version_minor              = "09"
+config_vm_version_patch              = "7"
 config_vm_data_year                  = "23"
-config_vm_data_month                 = "06"
+config_vm_data_month                 = "09"
 config_vm_data_iteration             = "0"
 
 // --- OpenAI API configuration --- //
-config_openai_api_docker_image_version = "0.0.9"
+config_openai_api_docker_image_version = "0.0.10"
 
 // --- DNS configuration --- //
 config_dns_platform_subdomain     = "partner-platform"
 config_dns_subdomain_prefix       = "dev"
 config_dns_platform_api_subdomain = "api"
 
 // --- Web App configuration --- //
-config_webapp_release = "v0.4.11"
+config_webapp_release = "v0.5.11"
```

Comment:
I think we have 0.5.12 in PPP production, would you mind making sure that our development environments match production after a release?
Thanks!

Reply:
Yes, The latest webapp version(0.5.12) has been created after the production deployment(PPP and Platform) and I didn't deploy it to Dev yet.
I'll do it now on. Thanks for the comment.